src/AddTodoForm.js

```diff
@@ -26,7 +23,6 @@ function AddTodoForm ({onAddTodo}) {
 <h1>  Form </h1>
 <form onSubmit={handleAddTodo}>
 
-
 <InputWithLabel name ='todoTitle' value={todoTitle}
 onChange={handleTitleChange} autoFocus={true}>
```

Comment:
I cant see todos,
So you can try to change 
`value={todoTitle}`
`onChange={handleTitleChange}` 

to

`todoTitle={todoTitle}`
`handleTitleChange={handleTitleChange}`

and check if it works